sources/packages/web/src/views/aest/student/applicationDetails/AssessmentsSummary.vue

```diff
@@ -20,6 +20,12 @@
       @viewApplicationException="goToApplicationException"
       @viewOfferingRequest="goToOfferingRequest"
     />
+    <manual-reassessment
+      class="mb-5"
+      :applicationId="applicationId"
+      @reassessmentTriggered="reloadComponents"
+      :key="manualReassessmentKey"
```

Comment:
Probably `manualReassessmentKey` will no longer be needed once the criteria is based on the original assessment. Does it make sense?

Reply:
I couldn't follow the purpose of `manualReassessmentKey`. But as it seems to be something to be removed I am not going to ask for details.